lms/djangoapps/ccx/views.py

```diff
@@ -433,9 +440,17 @@ def ccx_invite(request, course, ccx=None):
             email = user.email
         try:
             validate_email(email)
-            course_key = CCXLocator.from_course_locator(course.id, ccx.id)
-            email_params = get_email_params(course, auto_enroll, course_key=course_key, display_name=ccx.display_name)
-            if action == 'Enroll':
+        except ValidationError:
+            log.info('Invalid user name or email when trying to invite students: %s', email)
+            continue
+
+        email_params = get_email_params(course, auto_enroll, course_key=course_key, display_name=ccx.display_name)
+        if action == 'Enroll':
+            # check if there are spots left
+            if CourseEnrollment.objects.is_course_full(ccx_course_overview):
```

Comment:
@giocalitri  i think you need to update logic of ```ccx_invite```, 
If ```is_course_full is True``` then there is no point of validate student , just check in start of loop if  mode is ```enroll``` and  ```is_course_full is True``` then inform coach that he cannot add student.